README.rst

```diff
@@ -802,6 +802,17 @@ homebrew and try again:
 
 Once installed then repeat the `Installation process <#windows-linux-macos-with-pip>`_
 
+2. **CONNECTING TO MAC SERVICES**
+
+If you running services on your Mac, for example elasticsearch, and having trouble connecting then make sure your host is set to "docker.for.mac.localhost:PORT". In this example I set it to "docker.for.mac.localhost:9200" for the ES you are talking to. Example:
```

Comment:
I am hesitant on this change. It is very specific to mac and leaves the other OS high and dry. It seems that docker-for-mac and docker-for-windows supports `host.docker.internal` with  v18.03 or higher[1]. Looks like docker-for-linux is [working on the support](https://github.com/docker/for-linux/issues/264) as well. 

This also feels like just re-documenting docker to a point. These details (ideally) should be hidden away from customers but that's not always reasonable or possible.

With that said, I think we should make this abstract to all OS's by linking out to dockers docs on this instead of just mac. Thoughts?

Can you remove the "In this example I set it to "docker.for.mac.localhost:9200" for the ES you are talking to." and just have the `Example:`

[1] : https://stackoverflow.com/questions/31324981/how-to-access-host-port-from-docker-container/43541732#43541732